client/call_fn.go

```diff
@@ -47,7 +47,11 @@ func CallFN(u string, content io.Reader, output io.Writer, method string, env []
 		return fmt.Errorf("error running route: %s", err)
 	}
 
-	req.Header.Set("Content-Type", "application/json")
+	if contentType != "" {
+		req.Header.Set("Content-Type", contentType)
+	} else {
+		req.Header.Set("Content-Type", "text/plain")
```

Comment:
I don't have a strong argument for or against defaulting to anything vs nothing I suppose, other than leaving it unset feels... a bit like going out and then having that nagging feeling that you'd forgotten something.  True that a simple curl would leave it unset unless explicitly stated.  I wonder of any of the FDKs are expecting it be set to *something*? @shaunsmith specifically requested this change initially because the Node FDK checks Content-Type to decide whether to json-parse the body first or not before passing it on.

Reply:
def need a way to set it, at least. defaulting to text/plain instead of application/json should gleefully bust anyone relying on this behavior which makes me happy, and it seems a saner default at least (everything is text, not everything is json). hopefully FDKs aren't enforcing it to be set, just doing certain things for certain values if set -- some clients may not set this and it probably shouldn't cause issues (content length has/had a similar issue)